safe_transaction_service/utils/tasks.py

```diff
@@ -80,9 +79,7 @@ def only_one_running_task(
     lock_name = f"locks:tasks:{task.name}"
     if lock_name_suffix:
         lock_name += f":{lock_name_suffix}"
-    with redis.lock(
-        lock_name, blocking_timeout=blocking_timeout, timeout=lock_timeout
-    ) as lock:
+    with redis.lock(lock_name, blocking=False, timeout=lock_timeout) as lock:
```

Comment:
Remove from function documentation the `blocking_timeout` https://github.com/safe-global/safe-transaction-service/pull/1574/files#diff-c9cec6c9cc96fbb7b10fac3117122b25c4107e0d2f402d7135f5f4c5b80732b6R68

Reply:
Done